tests/system/libraries/SystemTestSpy/speechSpyGlobalPlugin.py

```diff
@@ -565,7 +565,7 @@ def terminate(self):
 		self._server.stop()
 
 
-def _crashNVDA():
+def _crashNVDA(param: int = 0):

```

Comment:
Can you add a docstring for this param? Or perhaps use a lambda or wrap the call to this to adapt it to the `queueAsApc` requirements.

Reply:
Done.